mobile/android-deploy-app/action.yml

```diff
@@ -59,6 +59,9 @@ inputs:
   tag_suffix:
     description: The tag suffix that should be used 
     required: optional
+  changes_not_sent_for_review:
+    description: The changesNotSentForReview flag for Google Play Console upload
+    required: true
```

Comment:
instead of making required I would make this optional but have default

Reply:
@pkramaric alright, I will make default as true then

Reply:
@pkramaric PR updated